pkg/util/signals/signal_posix.go

```diff
@@ -1,3 +1,4 @@
+//go:build !windows
```

Comment:
will this line conflict the the line below?

Reply:
This marker is automatically added by `go fmt`.

ref: https://go.googlesource.com/proposal/+/master/design/draft-gobuild.md#design

Reply:
> //go:build lines
> Go 1.17 introduced //go:build lines as a more readable way to write build constraints, instead of // +build lines. As > of Go 1.17, gofmt adds //go:build lines to match existing +build lines and keeps them in sync, while go vet > diagnoses when they are out of sync.
> 
> Since the release of Go 1.18 marks the end of support for Go 1.16, all supported versions of Go now understand > //go:build lines. In Go 1.18, go fix now removes the now-obsolete // +build lines in modules declaring go 1.18 or later > in their go.mod files.
> 
> For more information, see https://go.dev/design/draft-gobuild.

https://tip.golang.org/doc/go1.18#go-command

Reply:
got it.